email/smtp.go

```diff
@@ -57,7 +57,7 @@ func (cfg *SmtpEmailerConfig) UnmarshalJSON(data []byte) error {
 	if smtpCfg.Host == "" {
 		return errors.New("must set SMTP host")
 	}
-	if smtpCfg.Port == "" {
+	if smtpCfg.Port == 0 {
```

Comment:
How did this compile? Maybe it would be worth adding a test here? (Also, please ask @bobbyrullo to take a look)